messages/previewMessages.md

```diff
@@ -0,0 +1,39 @@
+# conflicts.none
+
+No conflicts found.
+
+# conflicts.header
+
+Conflicts [%s] Run the command with --ignore-conflicts to override
```

Comment:
```suggestion
Conflicts [%s]. Run the command with --ignore-conflicts to override.
```

Reply:
Okay?  Not sure if that first period makes sense because I can't actually picture it.

Reply:
it's gonna be a number, so there'll be a table header that says `Conflicts [5]. Run the command with --ignore-conflicts to override`

All the tables have the same format.